server/controller/routes/admin/viewEventsOnDate.js

```diff
@@ -0,0 +1,17 @@
+const moment = require('moment');
+
+const { getEventsDate } = require('../../../database/queries/events');
+
+const today = moment().format('YYYY-MM-DD');
+const viewEventsOnDate = (req, res, next) => {
+  getEventsDate().then(({ rows }) => {
+    if (rows.length !== 0) {
```

Comment:
Very small style thing here, I will normally try to deal with the error case first and return early - it will reduce the number of things you need to keep in your mind when you read the code, for example instead of 

```
if (rows.length !== 0) {
  // do something
  // do something else
  // ...
} else {
  // handle the case we specified a while ago
}
```

you could do

```
if (rows.length === 0) {
  return res.json({ events: 'no events...' })
}

// then here you've already dealt with the error case and don't have to think about it any more

```